### ISSUE TITLE
ErrorException because of number of elements

### ISSUE THREAD

**Issue by marflow** [goellner/statamic-data-importer#2]
Hello,

 I´m trying to use the addon but i get the error 
```
ErrorException in DataImporterController.php line 89:
array_combine(): Both parameters should have an equal number of elements
```
Not sure what that means. I have more fields in the fieldset than in the table I wish to import, but I not getting far enough to choose a colletion. There´s one taxonomies field in the table, and the values are seperated by "|".

Thanks a lot for a hint what may be wrong :)

-> One thing came to mind right now: I have ";" as seperators, I will check for it in the actual data. Maybe this is the problem. … No success unfortunately.

**Comment by bassigbc** [comment]
I have the same problem, any news? Thanks

**Comment by bassigbc** [comment]
It seems that the ADDON doesn't recognize the difference between newlines at the end of the row and newlines inside text fields, so If there are fields with newline chars the PHP report the array_combine error

**Comment by goellner** [comment]
I have no time currently to fix this issue, but I am open to pull requests

**Comment by goellner** [comment]
@bassigbc fixed in 2.0.0 release. please let me know, if you experience any more issues when importing data
